core/statedb/state_cache.go

```diff
@@ -24,13 +24,18 @@ type StateCache struct {
 	PendingOnChainOperationsHash    []byte
 	Txs                             []*tx.Tx
 
-	// Updated in executor's ApplyTransaction method.
+	// Record the flat states that should be updated.
 	PendingNewAccountIndexMap      map[int64]int
 	PendingNewLiquidityIndexMap    map[int64]int
 	PendingNewNftIndexMap          map[int64]int
 	PendingUpdateAccountIndexMap   map[int64]int
 	PendingUpdateLiquidityIndexMap map[int64]int
 	PendingUpdateNftIndexMap       map[int64]int
+
+	// Record the tree states that should be updated.
+	DirtyAccountsAndAssetsMap map[int64]map[int64]bool
```

Comment:
Please use lower case.

Reply:
I will fix this in the next refactor tx PR, after this PR merged, I will gonna to rebase this, and fix this one.